BUILD.md

```diff
@@ -78,6 +78,8 @@ Or alternatively you can install QtCreator and then open the pianobooster.pro.
 
 **INSTALL_ALL_LANGS**: install all languages; disabled by default for cmake and always enabled for qmake.
 
+**USE_SYSTEM_RTMIDI**: build with system (not bundled) rtmidi; disabled by default.
```

Comment:
Please see my comment below about USE_BUNDLED_RTMIDI.

This text line could then read

****USE_BUNDLED_RTMIDI**: build with bundled (not system) only required for old distributions that do not include the system rtmidi; disabled by default.

Reply:
Undeleting my comment above which read.
Please see my comment above about USE_BUNDLED_RTMIDI.

This text line could then read

**USE_BUNDLED_RTMIDI** : build with bundled (not system) rtmidi. This is only required for older distributions that do not include the system rtmidi; disabled by default.